styles/globals.css

```diff
@@ -21,3 +21,9 @@ ds-container{
   padding-left: 26px;
   padding-right: 26px
 }
+
+.markdown_div ul {
```

Comment:
For formatting text within Markdown components

Reply:
Please don't put styles in globals.css if possible. Tailwind typography hopefully will work instead

Reply:
Updated to format using tailwind.